pkg/apis/onmetal/types_cloudprofile.go

```diff
@@ -48,8 +48,21 @@ type StorageClass struct {
 	Name string
 	// Type is referring to the VolumeClass to use for this StorageClass
 	Type string
+	// ResizePolicy describes the supported expansion policy of a VolumeClass.
+	// If not set default to Static expansion policy.
+	ResizePolicy ResizePolicy
```

Comment:
@afritzler  we already have Type(Which is nothing but volumeClass) field in storage class and volume class already has the ResizePolicy field which decides if it's expandOnly or static, so adding new ResizePolicy field here seems redundant.

Reply:
Changed logic to read resize policy from VolumeClass and set expandable property in StorageClass accordingly